### ISSUE TITLE
How to initiate?

### ISSUE THREAD

**Issue by jonasostrom** [NextLevel/NextLevel#17]
I get these error-messages when trying to implement NextLevel in my viewcontroller

- Cannot assign value of type 'CreateAccountViewController' to type 'NextLevelDelegate?'
- Value of type 'NextLevel' has no member 'photoDelegate'
- Value of type 'CreateAccountViewController' has no member 'previewView'

NextLevel.sharedInstance.start()
gives me:
- Call can throw, but it is not marked with 'try' and the error is not handled

```
import NextLevel


class CreateAccountViewController: UIViewController, UINavigationControllerDelegate {

  
    override func viewDidLoad() {
        super.viewDidLoad()
        NextLevel.sharedInstance.delegate = self
        NextLevel.sharedInstance.photoDelegate = self
        setupCameraPreview()
    }

    func setupCameraPreview(){
        let screenBounds = UIScreen.main.bounds
        self.previewView = UIView(frame: screenBounds)
        if let previewView = self.previewView {
            previewView.autoresizingMask = [.flexibleWidth, .flexibleHeight]
            previewView.backgroundColor = UIColor.black
            NextLevel.sharedInstance.previewLayer.frame = previewView.bounds
            previewView.layer.addSublayer(NextLevel.sharedInstance.previewLayer)
            self.view.addSubview(previewView)
        }
    }

```
Is there some example-code how to implement?

**Comment by piemonte** [comment]
Hey @jonasostrom – the sample project provides some examples on use. `master` adds more delegates than what is available in the `0.0.4` release, just depends on how you're installing. sometime today, i'll be pushing `master` as `0.1.0`.

check out these resources:

0.0.4: https://github.com/NextLevel/NextLevel/blob/0.0.4/Project/NextLevel/CameraViewController.swift

master: https://github.com/NextLevel/NextLevel/blob/master/Project/NextLevel/CameraViewController.swift

**Comment by jonasostrom** [comment]
i'm using 0.0.4
